owncloudApp/src/main/java/com/owncloud/android/presentation/viewmodels/settings/SettingsPictureUploadsViewModel.kt

```diff
@@ -161,14 +161,13 @@ class SettingsPictureUploadsViewModel(
         sourcePath: String? = _pictureUploads.value?.sourcePath,
         behavior: FolderBackUpConfiguration.Behavior? = _pictureUploads.value?.behavior,
         timestamp: Long? = _pictureUploads.value?.lastSyncTimestamp
-    ): FolderBackUpConfiguration =
-        FolderBackUpConfiguration(
-            accountName = accountName ?: accountProvider.getCurrentOwnCloudAccount()!!.name,
-            behavior = behavior ?: FolderBackUpConfiguration.Behavior.COPY,
-            sourcePath = sourcePath.orEmpty(),
-            uploadPath = uploadPath ?: PREF__CAMERA_UPLOADS_DEFAULT_PATH,
-            wifiOnly = wifiOnly ?: false,
-            lastSyncTimestamp = timestamp ?: System.currentTimeMillis(),
-            name = _pictureUploads.value?.name ?: pictureUploadsName
-        )
+    ): FolderBackUpConfiguration = FolderBackUpConfiguration(
+        accountName = accountName ?: accountProvider.getCurrentOwnCloudAccount()!!.name,
+        behavior = behavior ?: FolderBackUpConfiguration.Behavior.COPY,
+        sourcePath = sourcePath.orEmpty(),
+        uploadPath = uploadPath ?: PREF__CAMERA_UPLOADS_DEFAULT_PATH,
+        wifiOnly = wifiOnly ?: false,
+        lastSyncTimestamp = timestamp ?: System.currentTimeMillis(),
+        name = _pictureUploads.value?.name ?: pictureUploadsName
+    )
```

Comment:
It could be helpful to add a log here to know exactly the new configuration. What do you think?

```suggestion
    ): FolderBackUpConfiguration = FolderBackUpConfiguration(
        accountName = accountName ?: accountProvider.getCurrentOwnCloudAccount()!!.name,
        behavior = behavior ?: FolderBackUpConfiguration.Behavior.COPY,
        sourcePath = sourcePath.orEmpty(),
        uploadPath = uploadPath ?: PREF__CAMERA_UPLOADS_DEFAULT_PATH,
        wifiOnly = wifiOnly ?: false,
        lastSyncTimestamp = timestamp ?: System.currentTimeMillis(),
        name = _pictureUploads.value?.name ?: pictureUploadsName
    ).also {
        Timber.d("Picture uploads configuration updated. New configuration: $it")
    }
```

Reply:
Great proposal, I do not use `.also{}` normally, but here is perfect.